modules/serverless-domain-mapping-custom-tls-cert.adoc

```diff
@@ -26,6 +16,20 @@ You can add an existing TLS certificate with a `DomainMapping` custom resource (
 ----
 $ oc create secret tls <tls_secret_name> --cert=<path_to_certificate_file> --key=<path_to_key_file>
 ----
++
+[IMPORTANT]
+====
+If you are using {SMProductName} as the ingress for your {ServerlessProductName} installation, or you are using a third party secret provider such as cert-manager, label this secret with the following `<key>:<value>` pair:
+
+[source,yaml]
+----
+“networking.internal.knative.dev/certificate-uid": “<value>”
+----
+
+Cert-manager users can use the secret template offered to automatically generate secrets with the correct label.
+
+In this case, secret filtering is done based on the key only, but this value can carry useful information such as the certificate ID that the secret contains.
```

Comment:
[Side note cert manager is tp on OCP](https://docs.openshift.com/container-platform/4.10/security/cert_manager_operator/cert-manager-operator-install.html).
I am wondering if we should test to verify that the secret template is available.

Reply:
Again this would be a question for QE probably to confirm before we publish the docs. We can add the note about it being tech preview to our docs as well, though.

Reply:
I have verified using the cert-manager's Certificate
```  
secretTemplate:
    labels:
      networking.internal.knative.dev/certificate-uid: example.com
```
works. (the redhat cert-manager operator is still tech preview though)